docs/api2.yaml

```diff
@@ -283,9 +283,391 @@ paths:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: fail
+                  data:
+                    type: string
+                    example:
+                      crowdactionID: crowdaction does not exist
+      security:
+        - FirebaseAuthorizer: []
+    delete:
+      tags:
+        - Crowdaction
+      summary: Delete a specific comment on a specific crowdaction
+      parameters:
+        - $ref: "#/components/parameters/ApiVersionParameter"
+        - name: crowdactionID
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: commentID
+          in: path
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: password
+          description: Only include if crowdaction requires password
+          required: false
+          schema:
+            type: string
+            format: password
+      responses:
+        "200":
+          description: Deleted
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: success
+                  data:
+                    default: null
+        "401":
+          description: Unauthorized (Invalid password)
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: fail
+                  data:
+                    type: string
+                    example:
+                      password: Invalid or missing password
+        "400":
+          description: Bad request (Tried to delete foreign comment)
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: fail
+                  data:
+                    type: string
+                    example:
+                      userID: You can only delete your own comments
+        "403":
+          $ref: "#/components/responses/UnsupportedClientVersion"
+        "404":
+          description: Crowdaction or comment not found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: fail
+                  data:
+                    type: string
+                    example:
+                      crowdactionID: crowdaction does not exist
+      security:
+        - FirebaseAuthorizer: []
+  "/crowdactions/{crowdactionID}/comments/{commentID}/report":
+    post:
```

Comment:
We also need a `reason` here. It's not a reason they type, it's something we have statically declared.
<details>
  <summary>Image for example</summary>
  
  ![image](https://user-images.githubusercontent.com/42929161/154818439-5c8bfdcc-852f-48be-af18-edfa2621e367.png)
  
</details>